.rubocop_todo.yml

```diff
@@ -292,6 +284,13 @@ Style/IndentHash:
   Enabled: false
 
 # Offense count: 1
+# Cop supports --auto-correct.
+# Configuration parameters: Width.
+Style/IndentationWidth:
+  Exclude:
+    - 'lib/prison_schedule.rb'
```

Comment:
Oops.  Missed that.  Sec.


On 25 September 2015 at 16:54, Steve Marshall <notifications@github.com>
wrote:

> In .rubocop_todo.yml
> <https://github.com/ministryofjustice/prison-visits/pull/328#discussion_r40444929>
> :
>
> > @@ -292,6 +284,13 @@ Style/IndentHash:
> >    Enabled: false
> >
> >  # Offense count: 1
> > +# Cop supports --auto-correct.
> > +# Configuration parameters: Width.
> > +Style/IndentationWidth:
> > +  Exclude:
> > +    - 'lib/prison_schedule.rb'
>
> You still need to fix the one-space indent in prison_schedule.rb, I think?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ministryofjustice/prison-visits/pull/328/files#r40444929>
> .
>